### ISSUE TITLE
Fix needed [Code] error in command to install c-lightning prerequisites

### ISSUE THREAD

**Issue by vnprc** [lnbook/lnbook#316]
**Describe the error**
In the node_client chapter the apt-get command to install c-lightning prerequisites is missing a newline character and fails to find libsodium-dev
```
$ sudo apt-get install -y \
  autoconf automake build-essential git libtool libgmp-dev \
  libsqlite3-dev python python3 python3-mako net-tools zlib1g-dev \ libsodium-dev gettext
```

**To Fix**
```
$ sudo apt-get install -y \
  autoconf automake build-essential git libtool libgmp-dev \
  libsqlite3-dev python python3 python3-mako net-tools zlib1g-dev \
  libsodium-dev gettext
```

**Comment by vnprc** [comment]
https://github.com/lnbook/lnbook/pull/319

**Comment by aantonop** [comment]
Fixed by #319
